pkg/tbtc/node.go

```diff
@@ -205,3 +268,136 @@ func (n *node) joinDKGIfEligible(seed *big.Int, startBlockNumber uint64) {
 		logger.Infof("not eligible for DKG with seed [0x%x]", seed)
 	}
 }
+
+// decideMemberFate decides what the member will do in case it failed to publish
+// its DKG result. Member can stay in the group if it supports the same group
+// public key as the one registered on-chain and the member is not considered as
+// misbehaving by the group.
+func (n *node) decideMemberFate(
```

Comment:
Let's be specific: s/decideMemberFate/decideSigningGroupMemberFate. Also, let's remember about the change in the comment.

Reply:
Done in 0bfc371f27dfbdf88d59e76e244a06f03dad8422.